src/protocol/sort/bit_permutations.rs

```diff
@@ -46,7 +47,7 @@ impl<'a, F: Field> BitPermutations<'a, F> {
     #[allow(dead_code)]
     pub async fn execute(
         &self,
-        ctx: ProtocolContext<'_, F>,
+        ctx: ProtocolContext<'_, Replicated<F>, F>,
```

Comment:
This is the first one I want to modify to use SecretShare<F> instead of Replicated.